src/sequence.rs

```diff
@@ -0,0 +1,155 @@
+//! This module manages project sequences
+extern crate sfml;
```

Comment:
Move this into the top level module `lib.rs`. It should be consistent with git2 and rust-serialize.